### DISCUSSION TITLE
What is the difference between Minio Helm Chart and Operator?

### DISCUSSION THREAD

**Discussion by faxenmaxen** [minio/minio#11679] [Q&A] [answered]
Hello,

- What is the difference between Minio Helm Chart and Operator?
- What can I do with the operator that Helm Chart cannot do?

**Comment by harshavardhana** [comment, marked as answer]
The operator is an idiomatic k8s lifecycle manager for your services, helm charts are nothing but a script to generate k8s YAMLs.  We recommend using operator.
